lib/src/game/game_widget.dart

```diff
@@ -423,65 +425,72 @@ Widget _applyBasicGesturesDetectors(Game game, Widget child) {
 
 Widget _applyAdvancedGesturesDetectors(Game game, Widget child) {
   final Map<Type, GestureRecognizerFactory> gestures = {};
-  final List<_GenericTapEventHandler> tapHandlers = [];
-  final List<_GenericDragEventHandler> dragHandlers = [];
-
-  if (game is HasTapableComponents) {
-    tapHandlers.add(_GenericTapEventHandler()
-      ..onTapDown = game.onTapDown
-      ..onTapUp = game.onTapUp
-      ..onTapCancel = game.onTapCancel);
-  }
-
-  if (game is MultiTouchTapDetector) {
-    tapHandlers.add(_GenericTapEventHandler()
-      ..onTapDown = game.onTapDown
-      ..onTapUp = game.onTapUp
-      ..onTapCancel = game.onTapCancel);
+  int lastGenDragId = 0;
```

Comment:
if the tree gets rebuilt everything gets restarted so I think it's ok to reset this. but I can move it to the global scope if needed.

Reply:
lastGeneratedDragId